### DISCUSSION TITLE
How to export the matrix name@assays[["RNA"]]@scale.data ?

### DISCUSSION THREAD

**Discussion by Lea2609** [satijalab/seurat#7144] [Q&A]
Hello everybody, 
I used Seurat and i would like to know how i can export the result of the function ScaleData ? 

I try 
test=name@assays[["RNA"]]@scale.data

But when i run str(test) the result is: 

 num [1:24333, 1:33968] -0.0613 -0.0242 -0.1215 -0.0108 -0.0359 ...
 - attr(*, "dimnames")=List of 2
  ..$ : chr [1:24333] ...
  ..$ : chr [1:33968] ...

I would like remove the attr 

thank's

**Comment by wudustan** [comment]
Hi, better way of explorting this using native Seurat functionality:

`scale.data.mat <- GetAssayData(seurat.object, slot = "scale.data", assay = "RNA")`

This will give you a sparse matrix of your scaled data. What kind of format do you need the data to be in?

**Reply by Lea2609** [reply]
Thanks, I needed it to be in the matrix form and thought that in the form: matrix array was giving me errors for another function but not.
